torch/_torch_docs.py

```diff
@@ -7951,10 +7951,17 @@ def merge_dicts(*dicts):
 dimension :attr:`dim` is input from :attr:`start` to ``start + length``. The
 returned tensor and :attr:`input` tensor share the same underlying storage.
 
+:attr:`start` accepts a `Tensor` for models that have it as such before calling
+`narrow`.  This avoids the call to `item()` for backends where it is expensive
+(like XLA).
+
 Args:
     input (Tensor): the tensor to narrow
-    dim (int): the dimension along which to narrow
-    start (Tensor or int): the starting dimension
+    dim (int): the dimension along which to narrow. Can be negative, which means
```

Comment:
Do we typically inform users that the dimension can be negative, or is it usually just assumed?